osf/models/schema_response.py

```diff
@@ -37,4 +52,141 @@ def all_responses(self):
     def updated_response_keys(self):
         '''Surfaces the keys of responses_blocks added in this revision.'''
         revised_keys = self.updated_response_blocks.values_list('schema_key', flat=True)
-        return list(revised_keys)
+        return set(revised_keys)
+
+    @classmethod
+    def create_initial_response(cls, initiator, parent, schema=None, justification=None):
```

Comment:
Minor: if this is exclusively for populating the initial set of responses when a registration is created, is `justification` necessary?